package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "dashboard",
```

Comment:
Project name is not corresponding, I recommend we use name of the project I have already used before,
I am not saying that mine is better 😺  just to be on the same page and it is not that important for this kind of project.